keys.txt

```diff
@@ -0,0 +1,30 @@
+e17be4a47a4e9c1e7a430a9b1538f6b5c9e315d25ae7f48a39c525010675a2bcc86172ce07475ed8791ed8bf802f64c1
```

Comment:
Same with this.

Reply:
these are some randomly generated key pairs that I used just so that anyone could play around with the project easily without having to generate their own keys...hopefully when we provide a real frontend for this project,  there will be no need for this